format-message.js

```diff
@@ -290,8 +290,8 @@ function formatNewsMessage(newsList, whoNumber) {
 
     if (description.length > 300) {
       end_index = description.indexOf(".", 100);
-      if (end_index < 0) {
-        end_index = 150;
+      if (end_index < 0 || end_index > 300) {
+        end_index = description.indexOf(" ", 250);
```

Comment:
Hopefully there's not a really long word there 😄  Could add another `if end_index > 300: end_index = 300` just to be safe, but this should work fine.

Reply:
I think this gives us enough breathing room that even if the word is supercalifragilistickexpialidotious we should still be ok